### ISSUE TITLE
FloatingEmojiPicker error

### ISSUE THREAD

**Issue by JimSangwine** [Foundry376/Mailspring#688]
##### What operating system are you using?
Manjaro i3

##### What version of Mailspring are you using?
1.1.4

**Bug?**
I pushed the down arrow when replying to a message. I had deleted the standard Mailspring sig - maybe that's related.

```
TypeError: Cannot read property 'data' of null
    at FloatingEmojiPicker (/src/components/composer-editor/emoji-plugins.jsx:77:18)
    at beginWork (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:133:310)
    at d (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:161:93)
    at f (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:161:417)
    at g (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:162:149)
    at m (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:169:173)
    at w (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:168:413)
    at batchedUpdates (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:171:357)
    at tc (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:54:269)
    at vd (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:71:249)
```

By the way - all these errors completely kill Mailspring. Selecting other messages does not replace the error. I have to do a `killall mailspring` and then reload the app.

##### Do you have any third-party plugins installed? If so, which ones?
No

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Not AFAIK

##### Is the issue reproducible with a particular attachment, message, signature, etc?

Sorry - I can't reproduce it.

--

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
No

**Comment by colinking** [comment]
I'm getting this error on Version 1.1.4 (1.1.4) on macOS 10.13.3, too.

**Comment by bengotow** [comment]
Hey! This was fixed in #685 and will ship in the next release—stay tuned.

**Comment by bengotow** [comment]
Hey folks—this has been fixed in 1.1.5 which shipped yesterday. You can download the update from https://getmailspring.com/download or choose "Check for Updates" within the app to install it. Enjoy!
